mobile/src/calendar/event-dialog/edit-sick-leave-event-dialog.tsx

```diff
@@ -31,14 +33,18 @@ class EditSickLeaveEventDialogImpl extends Component<EditSickLeaveEventDialogPro
     public render() {
         const selectedSickLeave = this.selectedSickLeave();
         const isEditable = selectedSickLeave ? !selectedSickLeave.calendarEvent.isCompleted : false;
+        let isDisabledForComplete = true;
```

Comment:
It's about button state not the sick leave - so it could be completed but only by user with proper permission, not for all. So IMO "canBeCompleted" is also could be confusing.